app/views/posts/index.html.erb

```diff
@@ -1,3 +1,20 @@
-<h1>Posts Page</h1>
+<div>
+  <%= render partial: "users/user_info", locals: {user: @user} %>
+
+  <div>
+    <% @posts.each do |post| %>
+      <%= render partial: "users/user_post_card", locals: {post: post} %>
+
+      <div class="comments">
+        <% post.most_recent_comments.each do |comment| %>
+          <p><%= comment.author.name %>: <%= comment.text %></p>
+        <% end %>
+      </div>
+    <% end %>
+  </div>
+</div>
```

Comment:
- [ ] Kindly align the `see all posts` to the middle as is in the given design

Given design
![image](https://user-images.githubusercontent.com/81932478/214692838-8ee77514-21eb-41f6-920b-8cd153592500.png)

Your design
![image](https://user-images.githubusercontent.com/81932478/214692894-5938b538-19ff-4aa0-a172-63fad1e80cf3.png)

Reply:
Thanks, @Munsa1 for the feedback.